codecov_cli/helpers/ci_adapters/cloudbuild.py

```diff
@@ -0,0 +1,42 @@
+import os
+
+from codecov_cli.helpers.ci_adapters.base import CIAdapterBase
+
+
+class GoogleCloudBuildCIAdapter(CIAdapterBase):
```

Comment:
CloudBuild is the name of the CI, so drop the extra `CI`
```suggestion
class GoogleCloudBuildAdapter(CIAdapterBase):
```

Reply:
Agreed and fixed.